platform/android/java/lib/src/org/godotengine/godot/Godot.kt

```diff
@@ -876,6 +924,40 @@ class Godot(private val context: Context) {
 		mClipboard.setPrimaryClip(clip)
 	}
 
+	fun ShowFilePicker(currentDirectory: String, filename: String, fileMode: Int, filters: Array<String>) {
```

Comment:
Similar comment as above, let's move this method and the one above into their own file. Also the method name should start with a lower-case.